modules/social_features/social_tagging/social_tagging.module

```diff
@@ -0,0 +1,230 @@
+<?php
+
+/**
+ * @file
+ * Contains social_tagging.module.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\Core\Render\Element;
+use Drupal\node\Entity\Node;
+
+/**
+ * Implements hook_help().
+ */
+function social_tagging_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the social_tagging module.
+    case 'help.page.social_tagging':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Content tagging module') . '</p>';
+      return $output;
+
+    case 'entity.taxonomy_vocabulary.overview_form':
+      /* @var \Drupal\taxonomy\Entity\Vocabulary $vocabulary */
+      $vocabulary = $route_match->getParameter('taxonomy_vocabulary');
+      if ($vocabulary->id() === 'social_tagging') {
+        return '<p><strong>' . t('Notice: Drag and drop has intentionally been disabled for this vocabulary.') . '</strong></p>';
+      }
+
+    default:
+  }
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function social_tagging_form_taxonomy_term_social_tagging_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
I think we should consider making this configurable with a setting (so it can be overridden in the distro) even if it is only a simple on/off toggle.